WordPress/gutenberg
# FocalPointPicker: Convert to TypeScript

⚠️ Merge after #43791

Part of #35744
In preparation for #38730

## What?

Converts FocalPointPicker to TypeScript. To limit the scope of this PR, tests have not been touched.

## Why?

Better docs and devex.

## Testing Instructions

`npm run storybook:dev` and see the FocalPointPicker stories in Docs view.

✅ Type checks pass
